README.md

```diff
@@ -1,193 +1,28 @@
 # Schema Standardization
 
-This documentation describes and explains the ReproNim schema specification.
+To see the documentation concerning the ReproNim schema specification [click]( ??? ).
 
-## 0.0: Contents
```

Comment:
Also just to make sure the content that you had put in the mkdocs was simply a copy-paste of the content of the readme.